src/Network/Network.Test/Network.Test.csproj

```diff
@@ -14,7 +14,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.4.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.8.1-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="20.0.0-preview" />    
```

Comment:
@sobaner it doesn't look this version has been published to NuGet, and there isn't a local `.nupkg` for this version -- how has this functionality been tested?

Reply:
@sobaner please add `nupkg` from `azure-powershell\tools\LocalFeed`